test/qtismtest/data/storage/xml/marshalling/OutcomeDeclarationMarshallerTest.php

```diff
@@ -58,25 +58,44 @@ public function testUnmarshallExternalScoredWithIllegalValue()
         $marshaller->unmarshall($element);
     }
 
-    public function testMarshallExternalScored()
+    /**
+     * @dataProvider qtiVersionsToTestForExternalScored
+     * @param $qtiVersion
```

Comment:
are those params required here? Isn't it enough that you use there dataProvider only? If so why don't u use type hinting here as well?

Reply:
Removed the @param.